PlusContent/privacy-mode.md

```diff
@@ -3,10 +3,10 @@
 **NOTE: This feature requires a [supporter certificate](https://sandboxie-plus.com/supporter-certificate/).**
 
 The concept of privacy mode and privacy enhanced (or Data Protection) boxes was introduced in **Sandboxie Plus v1.0.0**.
-In this mode, most of the locations on a PC are set to be treated like a Write[File/Key]Path, which means the sandboxed locations are writable, but the unsandboxed locations are not readable.
+In this mode, most of the locations on a PC cannot be read or written into, which helps protect you from viruses that steals your data since they can only steal the data in the sandbox.

```

Comment:
Could you incorporate `like a [Write[File/Key]Path](../Content/ResourceAccess.md)` somewhere in the sentence or in another sentence below? You're free to change the aforementioned wording in the code tags.

Reply:
Done!